any/attributeFromLargestArea.md

```diff
@@ -0,0 +1,62 @@
+# Attribute from Intersection
+
+This expression returns an attribute value from a featureSet where the largest feature intersects the input feature.
+
+## Use cases
+
+To suggest a value from a majority intersection. For instance, when drawing a polygon, return the county name for which the majority of the new polygon is in.
+
+## Workflow
+
+Copy and paste the expression found in the expression template below to the Arcade editor in ArcGIS Online, the relevant location in ArcGIS Pro, or the relevant location in a custom app.
+
+To configure the script to your layer, edit the first line to specify the layer you would like to use instead of the example layer. 
+
+```js
+var set = FeatureSetByName($datastore,"Building Footprints")
+```
+
+Also edit the last line to specify the attribute you would like to use intead of the example `objectid`.
+
+```js
+getAttributeFromLargestArea($feature, set, 'objectid')
+```
+
+## Expression Template
+
+This Arcade expression will extract the attribute value from a feature in a featureSet where the input feature intersection area is the largest.
+
+```js
+var set = FeatureSetByName($datastore, 'Building Footprints')
+
+function getAttributeFromLargestArea(feat, set, field) {
+    var items = intersects(set, feat)
+    console(items)
+
+    if (items == false) {
+        return { 'errorMessage': 'No intersection found' }
+    }
+
+    if (count(items) == 1) {
+        var result = first(items)
+
+        return result[field]
+    }
+
+    var largest = -1
```

Comment:
Not important since we're dealing with Area...but for other functions for getting the max value you can also use `-Infinity`. But since negative areas don't make sense, any number less than zero suffices.

Reply:
Neat. Does negative infinity and -1 use the same amount of memory?

Reply:
I believe so, but I'll need to double-check this.